test/parallel/test-dotenv-edge-cases.js

```diff
@@ -135,17 +135,39 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 0);
   });
 
-  it('should handle when --env-file is passed along with --', async () => {
-    const child = await common.spawnPromisified(
```

Comment:
Why did you remove this test?

Reply:
I didn't remove it, I just moved to the rest of the added tests.